requirements/base.txt

```diff
@@ -18,10 +18,11 @@ django-cachalot==2.1.0
 django-ckeditor==5.6.1
 django-cloudflare-push==0.2.0
 django-multiselectfield==0.1.8
-django-filter==1.1.0
+django-filter==2.1.0
```

Comment:
`djangorestframework-filters==0.11.1` is no yet compatible with `django-filter==2.1.0`. Can we stay on `1.1.0`?

Reply:
Is does work though...

Reply:
feel free to change. I was just happy to find a working combination after a while of trying, but that does not mean its the only working combination